google-cloud-datastore/src/main/java/com/google/cloud/datastore/TransactionExceptionHandler.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.datastore;
+
+import com.google.cloud.ExceptionHandler;
+import com.google.cloud.ExceptionHandler.Interceptor;
+
+public class TransactionExceptionHandler {
+  public static final Interceptor TRANSACTION_EXCEPTION_HANDLER_INTERCEPTOR =
+      new Interceptor() {
+        @Override
+        public RetryResult beforeEval(Exception exception) {
+          if (exception instanceof DatastoreException) {
+            DatastoreException e = getInnerException((DatastoreException) exception);
+            if (e.getCode() == 10 || e.getReason() == "ABORTED") {
+              return RetryResult.RETRY;
+            }
+          }
+          return Interceptor.RetryResult.CONTINUE_EVALUATION;
```

Comment:
Be consistent with including `Interceptor.` before `RetryResult` (same in `BaseService`)